modules/node/content/cms_node.e

```diff
@@ -78,6 +80,10 @@ feature -- Access
 		deferred
 		end
 
+	status: INTEGER
+			-- Associated status for the current node.
+			-- [{0,Not_Published}, {1, Published}, {2, Trash}]	
```

Comment:
the comment does not use the same value as in CMS_NODE_API
I would just write

[{CMS_NODE_API}.Not_Published}, {CMS_NODE_API}.Published, {CMS_NODE_API}.Trashed]